build/Dockerfile

```diff
@@ -1,8 +1,12 @@
 FROM quay.io/water-hole/ansible-operator
 
+COPY preprocess_template.py ${HOME}/
+COPY test_preprocess_template.py ${HOME}/
+RUN python ${HOME}/test_preprocess_template.py
```

Comment:
Uhm, why? We want to have tests and run them ourselves when developing, not in the image I believe.

Reply:
If it is only for developing purpose, then i will delete it